### PR TITLE
Refactor: Merge backup to jobs container

### PR DESCRIPTION
**Pull request by shikharish**
Solves part of #114.

### CODE REVIEW

**Comment by shikharish** [comment]
cc: @rajivharlalka @harshkhandeparkar @chirag-ghosh

**Review by rajivharlalka — DISMISSED** [review]

**Review by chirag-ghosh — DISMISSED** [review]

File: docker-compose.prod.yml [hunk 1]
```diff
@@ -2,11 +2,11 @@ include:
   - docker-compose.yml
 
 services:
-  backup:
-    build: "./backup"
-    networks:
-      mysql-network:
```
Comment (harshkhandeparkar) [inline docker-compose.prod.yml:8]: doesn't it need access to the mysql network?

**Review by harshkhandeparkar — CHANGES_REQUESTED** [review]
have you tested this?

File: docker-compose.prod.yml (hunk 1 above)
Comment (shikharish) [reply docker-compose.prod.yml:8]: backup and all works fine

Comment (shikharish) [reply docker-compose.prod.yml:8]: Ah no. It does need the network

Comment (shikharish) [reply docker-compose.prod.yml:8]: @harshkhandeparkar fixed

**Review by harshkhandeparkar — APPROVED** [review]

**Review by rajivharlalka — APPROVED** [review]

**Pull request merged**